hal/src/thumbv6m/sercom/v1/spi.rs

```diff
@@ -168,13 +168,21 @@ macro_rules! spi_master {
                 // reset the sercom instance
                 sercom.spi().ctrla.modify(|_, w| w.swrst().set_bit());
                 // wait for reset to complete
+                #[cfg(feature = "samd20")]
+                while sercom.spi().status.read().syncbusy().bit_is_set()
+                    || sercom.spi().ctrla.read().swrst().bit_is_set()
+                {}
```

Comment:
same syncbusy peripheral comment

Reply:
I think you missed SPI in your reg sync cleanup

Reply:
Yes, sorry I just pushed my work of tonight and I will continue tomorrow, there are still a few things I need to clean :)